src/data/markdown/docs/01 guides/02 Using k6/14 Scenarios.md

```diff
@@ -55,726 +57,30 @@ export let options = {
 
 </div>
 
-### Common configuration
+## Common options
```

Comment:
`Common options` at this moment looks out of scope.

A possible solution could be to introduce earlier `Executors` and next `Common options`. 

For example, we point that each `executor` has its own configuration options but there are a set of common options shared between all the executor types, and describe the `Common` options section.

Reply:
Switched places of the sections. Good catch!